src/App.tsx

```diff
@@ -16,35 +20,69 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.Default);
+  const [isReverse, setReverse] = useState(false);
+  const visibleGoods = sortGoods(goodsFromServer, sortField, isReverse);
+
   return (
     <div className="section content">
       <div className="buttons">
-        <button type="button" className="button is-info is-light">
+        <button
+          type="button"
+          className={classNames('button is-info', {
+            'is-light': sortField !== SortType.ALPHABET,
+          })}
+          onClick={() => {
+            setSortField(SortType.ALPHABET);
+          }}
```

Comment:
The function name for the event handler should start with a verb to describe what it does. Consider renaming it to reflect the action it performs, like `handleAlphabetSort`.